src/App.tsx

```diff
@@ -1,204 +1,75 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTypes } from './types/FilterTypes';
+import { getPreparedTodos } from './utils/GetPrepatedTodos';
+import { getAmountOfActiveTodos } from './utils/getAmountOfActiveTodos';
+import { Footer } from './components/Footer';
+import { Errors } from './utils/Errors';
+import { ErrorsField } from './components/ErrorsField';
+import { Header } from './components/Header';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterInstructions, setFilterInstructions] = useState<FilterTypes>(
+    FilterTypes.All,
+  );
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.Default);
+  const [showErrorMessage, setShowErrorMessage] = useState<boolean>(false);
```

Comment:
1) If the default value in `useState` is a simple type, then `TypeScript` itself understands what should be in it
2) Why do we need this state? If we have an error, we can have the length of the error and output it
For example: 

`{errorMessage.length && <ErrorComponent />}`
```suggestion
  const [showErrorMessage, setShowErrorMessage] = useState(false);
```